### PR TITLE
minideck:0.2.1

### PR DESCRIPTION
**Pull request by knuesel**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: a package for dynamic slides (pause, uncover, ...) that aims for a simple interface, good performance and integration with fletcher and CeTZ.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

File: packages/preview/minideck/0.2.1/themes/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+The configuration function can accept additional parameters and return additional values in the output dictionary.
+The theme functions in the output dictionary can also accept non-standard parameters.
+
+The theme functions that make slides (`slide`, `title-slide` and possibly others) should be wrappers around the `slide` function given as input.
+
+For example an `ocean` theme from a `fancy-themes` package could be used like this:
+
+```typst
+import "minideck"
+import "fancy-themes"
```
Comment (elegaanz) [inline packages/preview/minideck/0.2.1/themes/README.md:26]: I don't know if it is that important given that this example is hypothetical anyway, but these two imports are not valid.

**Review by elegaanz — COMMENTED** [review]
Just a small comment, but maybe we can agree that it is not that important and just ignore it.

File: packages/preview/minideck/0.2.1/themes/README.md (hunk 1 above)
Comment (knuesel) [reply packages/preview/minideck/0.2.1/themes/README.md]: thanks for reviewing, the invalid package specs were intentional but the missing `#` was not. I pushed a change so it now says:

```typst
#import "@preview/minideck:0.2.1"
#import "@preview/fancy-themes:0.1.0" // hypothetical package
```

**Comment by knuesel** [comment]
Thanks for catching it (the missing `#` was definitely not intentional), it's fixed now.

**Comment by elegaanz** [comment]
Thank you :)

**Pull request merged**